linetools/spectra/io.py

```diff
@@ -538,6 +542,57 @@ def parse_linetools_spectrum_format(hdulist):
 
     return xspec1d
 
+def parse_hdf5(inp, **kwargs):
+    """ Read a spectrum from HDF5 written in XSpectrum1D format
+    Expects:  meta, data, units
+
+    Parameters
+    ----------
+    inp : str or hdf5
+
+    Returns
+    -------
+
+    """
+    import json
+    import h5py
+    # Open
+    if isinstance(inp, basestring):
+        hdf5 = h5py.File(inp, 'r')
+    else:
+        hdf5 = inp
+    # Data
+    data = hdf5['data'].value
```

Comment:
It seems more natural to me to have `hdf['flux']` and `hdf['wavelength']`, instead of `hdf['data']` containing both.